tests/unit/dataactvalidator/test_a1_appropriations.py

```diff
@@ -0,0 +1,58 @@
+from random import randint
+from tests.unit.dataactcore.factories.staging import AppropriationFactory
+from tests.unit.dataactcore.factories.domain import TASFactory
+from tests.unit.dataactvalidator.utils import number_of_errors, query_columns
+
+_FILE = 'a1_appropriations'
+
+def test_column_headers(database):
+    expected_subset = {"row_number", "allocation_transfer_agency", "agency_identifier", "beginning_period_of_availa",
+        "ending_period_of_availabil", "availability_type_code", "main_account_code", "sub_account_code"}
+    actual = set(query_columns(_FILE, database))
+    assert expected_subset <= actual
```

Comment:
I based this on the existing tests that have this test_column_headers.  That said, I haven't seen a case where I was expecting it to be a subset, so I'll go ahead and change these to test for equality.